### ISSUE TITLE
npm install chromedriver --chromedriver_version=LATEST broken with chromedriver 80.0.2

### ISSUE THREAD

**Issue by jazzzz** [giggio/node-chromedriver#260]
`npm install chromedriver@80.0.1 --chromedriver_version=LATEST` works, but `npm install chromedriver@80.0.2 --chromedriver_version=LATEST` does not. It throws this error :

```
> node install.js

internal/util.js:214
    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'original', 'function');
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "original" argument must be of type function
    at promisify (internal/util.js:214:11)
    at Object.<anonymous> (/home/user/project/node_modules/extract-zip/index.js:11:18)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/user/project/node_modules/chromedriver/install.js:14:20)
```

**Comment by giggio** [comment]
What SO are you on? What are the Node.js and Npm versions?

**Comment by jazzzz** [comment]
node v8.16.1, npm 6.14.3.

**Comment by giggio** [comment]
I noticed that this is a call to `stream.pipeline`. This function was added on Node.js 10, that is no longer supported. This lib stopped supporting it when Node.js stopped supporting it, and `80.0.2` updated `extract-zip` because of a vulnerability. The new version of `extract-zip` also dropped support for Node.js 8, and this is why you are seeing the error.

https://github.com/maxogden/extract-zip/blob/eb3c1edb8481bbf68da05cd3a824b1dcc697f908/index.js#L11

So, this scenario is not supported. To fix it you must upgrade to version 10 or later of Node.js.
